sdk/appcenter/src/main/java/com/microsoft/appcenter/ingestion/AppCenterIngestion.java

```diff
@@ -121,8 +121,10 @@ public ServiceCall sendAsync(String appSecret, UUID installId, LogContainer logC
         Map<String, String> headers = new HashMap<>();
         headers.put(INSTALL_ID, installId.toString());
         headers.put(APP_SECRET, appSecret);
-        if (mAuthToken != null && mAuthToken.length() > 0) {
-            headers.put(AUTHORIZATION_HEADER, String.format(AUTH_TOKEN_FORMAT, mAuthToken));
+        synchronized (this) {
```

Comment:
move it to a private method, synchronized block causes code coverage issue when used as a block within a method

Reply:
that is if we still have the issue with yellow synchronized covereage on local html report, please double check

Reply:
I don't see yellow coverage. All green.

Reply:
<img width="793" alt="screen shot 2019-02-19 at 2 24 38 pm" src="https://user-images.githubusercontent.com/623672/53051883-26893680-3452-11e9-8175-d4aeca79c124.png">

The ending curly brace is yellow (not talking about the if). Might be an issue after you fixed the if. If you have an issue with that one you will need a private method.

Reply:
Hmm on my mac it was an, i will look at it